database/tables/submissions.pg

```diff
@@ -9,7 +9,9 @@ columns
     format_errors: jsonb
     gradable: boolean default true
     graded_at: timestamp with time zone
-    grading_method: enum_grading_method
+    grading_method_external: boolean default false
```

Comment:
TO DO: Come back to this for review, as I don't believe this is used by anything. This information is available at the Question level and the grading method appears in the grading_jobs.

Reply:
Nope. Still needed for the question.js to determine the nature of the question on a submission. I suppose this could be altered to come directly from the Question file.